File: Php/app/Urls/CashgameEndUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class CashgameEndUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::cashgameEnd, $homegame);
		}

	}

}

File: Web/Models/UrlModels/CashgameEndUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{

	class CashgameEndUrlModel : HomegameUrlModel{

		public CashgameEndUrlModel(Homegame homegame)
            : base(RouteFormats.CashgameEnd, homegame)
        {
		}

	}

}